examples/run_field_dep_justMw.py

```diff
@@ -142,87 +142,28 @@
                 repetition_us=config_dict["repetition_us"],
                 tau_us=config_dict["tau_us"],
                 SW_kHz=config_dict["SW_kHz"],
-                amplitude=["amplitude"],
+                amplitude=config_dict["amplitude"],
                 ret_data=sweep_data,
             )
 sweep_data.set_prop("acq_params", config_dict.asdict())
 # }}}
 # {{{chunk and save data
-if phase_cycling:
-    sweep_data.chunk("t", ["ph1", "t2"], [4, -1])
-    sweep_data.setaxis("ph1", r_[0.0, 1.0, 2.0, 3.0] / 4)
-    if config_dict["nScans"] > 1:
-        sweep_data.setaxis("nScans", r_[0 : config_dict["nScans"]])
-    sweep_data.reorder(["ph1", "indirect", "t2"])
-    sweep_data.squeeze()
-    sweep_data.set_units("t2", "s")
-    fl.next("Raw - time")
-    fl.image(
-        sweep_data.C.mean("nScans")
-        .setaxis("indirect", "#")
-        .set_units("indirect", "scan #")
-    )
-    sweep_data.reorder("t2", first=False)
-    for_plot = sweep_data.C
-    for_plot.ft("t2", shift=True)
-    for_plot.ft(["ph1"], unitary=True)
-    fl.next("FTed data")
-    fl.image(
-        for_plot.C.mean("nScans")
-        .setaxis("indirect", "#")
-        .set_units("indirect", "scan #")
-    )
-else:
-    if config_dict["nScans"] > 1:
-        sweep_data.setaxis("nScans", r_[0 : config_dict["nScans"]])
-    sweep_data.rename("t", "t2")
-    fl.next("Raw - time")
-    fl.image(
-        sweep_data.C.mean("nScans")
-        .setaxis("indirect", "#")
-        .set_units("indirect", "scan #")
-    )
-    for_plot = sweep_data.C
-    for_plot.ft("t2", shift=True)
-    fl.next("FTed data")
-    fl.image(
-        for_plot.C.mean("nScans")
-        .setaxis("indirect", "#")
-        .set_units("indirect", "scan #")
-    )
+sweep_data.chunk("t", ["ph1", "t2"], [4, -1])
+sweep_data.setaxis("ph1", r_[0.0, 1.0, 2.0, 3.0] / 4)
+if config_dict["nScans"] > 1:
+    sweep_data.setaxis("nScans", r_[0 : config_dict["nScans"]])
```

Comment:
there's a shortcut for what you're doing here, which is `sweep_data.setaxis('nScans','#')`